__tests__/app.test.js

```diff
@@ -242,3 +240,61 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: responds with the posted comment", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        username: "icellusedkars",
+        body: "Hello",
+      })
+      .expect(201)
+      .then((res) => {
+        expect(res.body.data).toEqual({
+          comment_id: 19,
+          body: "Hello",
+          article_id: 1,
+          author: "icellusedkars",
+          votes: 0,
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("400: responds with 'bad request' if article_id is not an integer", () => {
+    return request(app)
+      .post("/api/articles/five/comments")
+      .send({
+        username: "icellusedkars",
+        body: "Hello",
+      })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+  test("404: responds with an error message when the endpoint is incorrect", () => {
+    return request(app)
+      .post("/api/articlez/5/comments")
+      .send({
+        username: "icellusedkars",
+        body: "Hello",
+      })
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toBe("Path not found");
+      });
+  });
+  test("404: responds with an error message when the endpoint is not found ", () => {
```

Comment:
A better description for this test would be something like "when the article_id doesn't exist" - it's a valid id, but there isn't an article with that ID *yet*.